src/main/webapp/login.html

```diff
@@ -1,22 +1,34 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>altruism-ator</title>
+    <title>Login</title>
     <link rel="stylesheet" href="style.css">
```

Comment:
It seems you didn't pull the basic code I uploaded on master, so the styling of the nav bar doesn't match.

Reply:
I think it's because your code is not merged with master yet, and I created my branch from master.